RickLeinecker/Fortuna
# Made Casus save and load automatically

**Description**
This PR makes casus code save and load to the database. If you want to test it, you have to edit the tankID in saveCasus and loadCasus.

The reason this is rediculously annoying is that functions aren't saved in JSON. So to make sure we keep the functions attached to every Casus block in the tree, we need to 'revive' it (technical word, I didn't make it up). That is what I spent most of the time doing here and what the ungodly switch statement is for.

There really isn't a better way of doing that par than this unfortunately, since each of the different blocks needs to correspond to its own object, so that it has its own evaluate method.



**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.

- Passes flow
- Saves and loads casus code. To verify, you need to
   - Create a new tank
   - Change the hardcoded id in saveCasus.js and loadCasus.js to the id of the tank you just created
   - then everything should work as you would expect

**Final Checklist:**
Go down the list and check them off.
